package.json

```diff
@@ -27,7 +27,8 @@
   },
   "homepage": "https://github.com/firstandthird/hapi-referrer#readme",
   "dependencies": {
-    "inbound": "0.0.6"
+    "referer-parser": "0.0.3",
+    "s-inbound": "github:PureClarity/inbound"
```

Comment:
Do we need this?

Reply:
Oh, weird that npm didn't remove it... I'll fix